### DISCUSSION TITLE
How to catch error when calling createUser in useDirectusAuth func in Nuxt 3

### DISCUSSION THREAD

**Discussion by chris-sab** [Intevel/nuxt-directus#224] [Q&A]
I did try to get error object when creating user in **Nuxt3**. I implemented the following logic but was not able to get error response.

```
<script setup>
const { createUser } = useDirectusAuth();
const register = async () => {
    try {
      await createUser(user.value);
    } catch (error) {
      console.log(error);
    }
  };
</script>
```
Here, I can not get `error` object when **createUser** api failed.

**Comment by sandros94** [comment]
Other than a missing `const user = ref()` in your `script setup`, could you check your console logs with something simpler like:
```vue
<script setup>
const { createUser } = useDirectusAuth();
const register = await createUser(user.value);
</script>
```

It is quite rare, but since the module already has a try-cache block it is possible that they are conflicting/overwriting in your particular instance.

Could you also share module, nuxt and server version?
